### DISCUSSION TITLE
Why isn't there a View component?

### DISCUSSION THREAD

**Discussion by OkancanCosar** [callstack/react-native-paper#4379] [General]
Why do we have to create a customized view component? Or is there a better way?

**Comment by gedu** [comment]
Hey hey, can you elaborate a bit more? What do you wanna achieve?

**Comment by OkancanCosar** [comment]
Actually what I was looking for was a customized canvas view. <PaperView> component that I can use instead of <View> without combining themes with React-navigation. This component has a basic view component that implements the background color of the paper theme.
But I just learned that when combined with the react-navigation theme, there is no need for this :)
